doc/api/deprecations.md

```diff
@@ -3318,6 +3318,22 @@ Type: Runtime
 might result in host name spoofing with unexpected input. These URLs will throw
 an error in future versions of Node.js, as the [WHATWG URL API][] does already.
 
+### DEP0171: Setters for `http.IncomingMessage` headers and trailers
+
+<!-- YAML
+changes:
+  - version:
+    - REPLACEME
```

Comment:
The YAML linter doesn't run on those, but if it did it would say `wrong indentation: expected 6 but found 4`
```suggestion
      - REPLACEME
```